### PR TITLE
Add node:logs command - closes #692

### LINKED ISSUE #692: Add node:logs command
### Description

`node:logs` command streams the log of particular locally installed instance.

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?
No implementation exists
### How did I fix it?
Implement the node logs feature
### How to test it?
`./bin/run node:logs --name=test-1.5`
### Review checklist

* The PR resolves #692 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: src/commands/node/logs.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }),
+  };
+
+  async run(): Promise<void> {
+    const { flags } = this.parse(LogsCommand);
+    const { name } = flags as Flags;
+
+    const { pm2_env } = await describeApplication(name);
+    const { pm_cwd: installDir, LISK_NETWORK: network } = pm2_env as Pm2Env;
+    const { logFileName }: NodeConfig = getNetworkConfig(installDir, network);
```
Comment (shuse2) [inline src/commands/node/logs.ts]: No need to parse the type here

File: src/commands/node/logs.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    const fName = `${installDir}/${logFileName}`;
+
+    const { stderr, stdout } = childProcess.spawn('tail', ['-f', fName]);
+
+    stdout.on('data', (data) => {
```
Comment (shuse2) [inline src/commands/node/logs.ts]: nice solution.
shouldn't we still listen to close event and clean up the stream and child process?

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: src/commands/node/logs.ts (hunk 2 above)
Comment (ManuGowda) [reply src/commands/node/logs.ts]: hmmm I will investigate this point 👍

File: src/commands/node/logs.ts (hunk 1 above)
Comment (ManuGowda) [reply src/commands/node/logs.ts]: why not use the type? any specific reason?

Comment (shuse2) [reply src/commands/node/logs.ts]: because it's returning the type `NodeConfig`, it's redundant

File: src/commands/node/logs.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	};
+
+	async run(): Promise<void> {
+		const { flags } = this.parse(LogsCommand);
+		const { name } = flags as Flags;
+
+		const { pm2_env } = await describeApplication(name);
+		const { pm_cwd: installDir, LISK_NETWORK: network } = pm2_env as Pm2Env;
+		const { logFileName } = getNetworkConfig(installDir, network);
+		const fName = `${installDir}/${logFileName}`;
```
Comment (shuse2) [inline src/commands/node/logs.ts]:
```suggestion
		const fileName = `${installDir}/${logFileName}`;
```

**Review by shuse2 — CHANGES_REQUESTED** [review]

**Review by shuse2 — APPROVED** [review]

**Pull request merged**